flake/templates.nix

```diff
@@ -5,6 +5,10 @@
       path = ../templates/simple;
       description = "A simple nix flake template for getting started with nixvim";
     };
+    new = {
+      path = ../templates/new;
+      description = "An experimental flake template for configuring nixvim directly with evalNixvim";
+    };
```

Comment:
How do we feel about the template name & description?

Reply:
Maybe we can mention flake-parts?